openedx/core/djangoapps/content/block_structure/config/__init__.py

```diff
@@ -10,12 +10,42 @@
 from .models import BlockStructureConfiguration
 
 # Switches
+# .. toggle_name: INVALIDATE_CACHE_ON_PUBLISH
+# .. toggle_implementation: WaffleSwitch
+# .. toggle_default: False
+# .. toggle_description: Invalidates block structure cache when changes to courses are published.
+# .. toggle_use_cases: opt_in
+# .. toggle_creation_date: 2017-02-23
+# .. toggle_tickets: https://github.com/edx/edx-platform/pull/14571
 INVALIDATE_CACHE_ON_PUBLISH = WaffleSwitch(
     "block_structure.invalidate_cache_on_publish", __name__
 )
+
+# .. toggle_name: STORAGE_BACKING_FOR_CACHE
+# .. toggle_implementation: WaffleSwitch
+# .. toggle_default: False
+# .. toggle_description: Enables retrieving block structures from storage instead of regenerating
```

Comment:
What about something like?:
```
When enabled, stores block structures in a more permanent storage like a database, that provides an additional backup for cache misses, instead of regenerating...
```